pam/internal/adapter/gdmmodel.go

```diff
@@ -271,3 +289,12 @@ func (m gdmModel) changeStage(s proto.Stage) tea.Cmd {
 		return nil
 	}
 }
+
+func (m gdmModel) stopConversations() gdmModel {
+	// We're about to exit: let's ensure that all the messages have been processed.
+	time.Sleep(gdmPollFrequency * 2)
```

Comment:
prefer time.After() then.

Reply:
It's twp different phases though, first I'd give gdm some time to complete the operations, then we force waiting. So first wait should still be with time.after()? sleep doesn't seem blocking other goroutines (as others are managed by bubbletea).

Reply:
Eh, also because using time after.... `S1037: should use time.Sleep instead of elaborate way of sleeping (gosimple)` (which was my guess too :))